src/Layout/Navbar.tsx

```diff
@@ -0,0 +1,72 @@
+import { makeStyles } from "@material-ui/core/styles";
+import { useTranslation } from "react-i18next";
+import Button from "@material-ui/core/Button";
+import { useEffect, useState } from "react";
+
+enum Languages {
+  Hindi = "hi",
+  Gujrati = "guj",
+  English = "en",
+}
+
+const useStyles = makeStyles({
+  root: {
+    width: "90%",
+    padding: "0.5rem 0",
+    paddingLeft: "35%",
+    margin: "auto",
+    backgroundColor: "grey",
+  },
+  button: {
+    margin: "10px",
+  },
+});
+
+const NavBar = () => {
+  const classes = useStyles();
+  const { i18n } = useTranslation();
+  const [currentLanguage, setCurrentLanguage] = useState(
+    process.env.DEFAULT_LANGUAGE !== undefined
+      ? process.env.DEFAULT_LANGUAGE
+      : Languages.English
```

Comment:
even if you don't write !==undefined ... it will work fine .... if it will get a DEFAULT_LANGUAGE in the env file it will take that otherwise from the Languages enum that you have defined !

Reply:
yes that is also  work fine. i updated that in new  commit.